### DISCUSSION TITLE
Please help the Homebrew people finalize their transmission-cli 4.0.0 formula!

### DISCUSSION THREAD

**Discussion by ilikepeaches** [transmission/transmission#4992] [Ideas]
### What is the issue?

I know that I'm making an issue on one project about a pull request on another, buuuut ... I wonder if anyone who's accustomed to building Transmission might be willing to stop by https://github.com/Homebrew/homebrew-core/pull/122688 and help them get their transmission-cli 4.0.0 formula working.  It's been two weeks, and they seem a little stalled.

Hopefully you will agree that the Transmission project has some interest in reaching out to help the Homebrew folks finish up — so that everyday Homebrew users can install the 4.0 version of transmission-daemon & etc.

Thanks!

### Which application of Transmission?

transmission-daemon

**Comment by ckerr** [comment]
CC @fxcoudert who is active in both projects

**Comment by ilikepeaches** [comment]
Still no 4.0.x formula, womp.  That pull request seems to have stalled again.

**Comment by ilikepeaches** [comment]
Now the pull request has been closed by its author, who says he doesn't have time to work on it further.

**Comment by fxcoudert** [comment]
Right now what is missing is a full source file: 4.0.0 and 4.0.1 are not present in https://github.com/transmission/transmission-releases

**Reply by ilikepeaches** [reply]
> Right now what is missing is a full source file: 4.0.0 and 4.0.1 are not present in https://github.com/transmission/transmission-releases

How do we go about making that happen?

**Reply by ckerr** [reply]
We're considering not using that repo anymore.

Can Homebrew use the release URLS instead, e.g. https://github.com/transmission/transmission/releases/download/4.0.1/transmission-4.0.1.tar.xz ?

**Reply by fxcoudert** [reply]
Should work if it contains all sources, including submodules. Second try: https://github.com/Homebrew/homebrew-core/pull/124844

**Reply by fxcoudert** [reply]
Shipped 🚀
